app/controllers/scim_rails/scim_groups_controller.rb

```diff
@@ -44,25 +44,25 @@ def create
 
       update_group_status(group) unless put_active_param.nil?
 
-      ScimRails.config.after_scim_response.call(group, "CREATED") unless ScimRails.config.after_scim_response.nil?
+      final_callback_hook(group, "CREATED")
```

Comment:
where are these methods coming from?
is this call still conditional?

Reply:
It's from `application_controller.rb`, I made it so it would check in the methods I made there instead